lib/auth/webauthn/login_mfa.go

```diff
@@ -0,0 +1,131 @@
+// Copyright 2022 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package webauthn
+
+import (
+	"context"
+	"errors"
+
+	"github.com/gravitational/teleport/api/types"
+	"github.com/gravitational/trace"
+
+	wantypes "github.com/gravitational/teleport/api/types/webauthn"
+)
+
+// LoginIdentity represents the subset of Identity methods used by LoginFlow.
+// It exists to better scope LoginFlow's use of Identity and to facilitate
+// testing.
+type LoginIdentity interface {
```

Comment:
Note to reviewers: most of the types and public methods in this file are the public bits of the old "login.go".